include/grabber/FramebufferWrapper.h

```diff
@@ -5,7 +5,7 @@
 
 ///
 /// The FramebufferWrapper uses an instance of the FramebufferFrameGrabber to obtain ImageRgb's from the
-/// displayed content. This ImageRgb is processed to a ColorRgb for each led and commmited to the
+/// displayed content. This ImageRgb is processed to a ColorRgb for each led and committed to the
 /// attached Hyperion.
```

Comment:
steinig mich bitte nicht, kann das nicht mit in Zeile 8? :D

Reply:
Done